awx/main/tasks/receptor.py

```diff
@@ -731,6 +769,3 @@ def remove_deprovisioned_node(hostname):
 
     # This will as a side effect also delete the InstanceLinks that are tied to it.
     Instance.objects.filter(hostname=hostname).delete()
-
-    # Update the receptor configs for all of the control-plane.
-    write_receptor_config.apply_async(queue='tower_broadcast_all')
```

Comment:
Why is this being taken out?  Nodes that previously linked to the decommissioned node will need to be updated.